kumascript/macros/HTTPSidebar.ejs

```diff
@@ -325,8 +325,8 @@ var text = mdn.localStringMap({
                 <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Headers/X-Content-Type-Options`, null, "X-Content-Type-Options")%></li>
                 <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Headers/X-Frame-Options`, null, "X-Frame-Options")%></li>
                 <li><%-web.smartLink(`/${locale}/docs/Web/HTTP/Headers/X-XSS-Protection`, null, "X-XSS-Protection")%></li>
-                <li><a href="https://wiki.mozilla.org/Security/Guidelines/Web_Security">Mozilla web security guidelines</a></li>
-                <li><a href="https://observatory.mozilla.org/">Mozilla Observatory</a></li>
+                <li><%-web.smartLink(`/${locale}/docs/Web/Security/Practical_implementation_guides`)%></li>
+                <li><a href="https://developer.mozilla.org/en-US/observatory">HTTP Observatory</a></li>
```

Comment:
If we decide to link to `/fr/observatory`, then we should use `${locale}` here as well, but I wouldn't recommend it.

Reply:
We went with linking to `/en-US/` only for now.